.zuul.yaml

```diff
@@ -25,6 +25,7 @@
             &imagetag "{{ zuul.tag is defined | ternary([zuul.get('tag', '').split('.')[0], '.'.join(zuul.get('tag', '').split('.')[:2]), zuul.get('tag', '')], ['latest']) }}"
           siblings:
             - github.com/opentelekomcloud/python-otcextensions
+    nodeset: ubuntu-focal
```

Comment:
I would honestly rather set nodeset for build-docker-image instead. We have same issue everywhere

Reply:
`build-docker-image` job came from opendev

Reply:
then we should define something like otc-build-docker-image in otc-zuul-jobs. I anyway wanted to also experiment with build-container-image instead and also trying to use buildah. This should be possible to achieve inside pod